modyn/trainer_server/internal/trainer/remote_downsamplers/deepcore_utils/README.md

```diff
@@ -0,0 +1,4 @@
+# DeepCore Utils
+The content of this folder is taken from DeepCore with minor adjustments.
+You can find the original code [here](https://github.com/PatrickZH/DeepCore/tree/main/deepcore/methods/methods_utils) and 
+the MIT license [here](https://raw.githubusercontent.com/PatrickZH/DeepCore/main/LICENSE.md)
```

Comment:
Similarily to the other time, we need to inline the license here (literally, add a new subheading and copy + paste the content of the LICENSE file here)